lib/resource/Organization.js

```diff
@@ -49,6 +49,30 @@ Organization.prototype.getDefaultGroupStore = function getDefaultGroupStore(/* [
   return this.dataStore.getResource(this.defaultGroupStoreMapping.href, args.options, OrganizationAccountStoreMapping, args.callback);
 };
 
+Organization.prototype.getDefaultAccountStore = function getDefaultAccountStore(/* [options,] callback */) {
+  var args = utils.resolveArgs(arguments, ['options', 'callback'], true);
+
+  return this.getDefaultAccountStoreMapping({ expand: 'accountStore' }, function (err, organizationAccountStoreMapping) {
+    if (err) {
+      return args.callback(err);
+    }
+
+    organizationAccountStoreMapping.getAccountStore(args.options, args.callback);
```

Comment:
There needs to be a null check here: `organizationAccountStoreMapping` will be `null` if there is no default account store mapping for the organization.  In this situation we need to callback with (null,null).  Right now it throws an exception because `getAccountStore` is undefined on null.  Back to @timothyej